src/gui/windows/main.zig

```diff
@@ -15,12 +15,15 @@ pub var window = GuiWindow {
 
 const padding: f32 = 8;
 
+fn exitGame(_:usize) void {
+	main.Window.deinit();
```

Comment:
This solution does not seem to work.
When I press the exit button it crashes with:
```
Segmentation fault at address 0x88
???:?:?: 0x163c44a in _glfwPollEventsX11 (/home/mint/.cache/zig/p/1220c46ebcac607065f942193b82a21b87213fca4b0d50c83461f8f5e22955fcb88f/src/x11_window.c)
/home/mint/Desktop/Cubyz/src/graphics/Window.zig:343:18: 0x1312194 in handleEvents (Cubyzig)
 c.glfwPollEvents();
                 ^
/home/mint/Desktop/Cubyz/src/main.zig:416:22: 0x130ae68 in main (Cubyzig)
  Window.handleEvents();
                     ^
/home/mint/Cubyz/compiler/zig/lib/std/start.zig:501:22: 0x130a579 in main (Cubyzig)
            root.main();
                     ^
../sysdeps/nptl/libc_start_call_main.h:58:16: 0x7fd96cd9ed8f in __libc_start_call_main (../sysdeps/x86/libc-start.c)
../csu/libc-start.c:392:3: 0x7fd96cd9ee3f in __libc_start_main_impl (../sysdeps/x86/libc-start.c)
???:?:?: 0x130a224 in ??? (???)
???:?:?: 0x0 in ??? (???)
```

Reply:
ah oops, didnt see the segfault under all of the thread sanitizers stuff.